packages/accordion/src/styled.tsx

```diff
@@ -0,0 +1,55 @@
+import colors from '@littlespoon/theme/lib/colors'
+import { p3, p4 } from '@littlespoon/theme/lib/fonts/paragraph'
+import { family, weight } from '@littlespoon/theme/lib/fonts/primary'
+import styled from 'styled-components'
+
+export const Wrapper = styled.div<{
```

Comment:
Can we rename this to `ContentWrapper` and move it below `Item`? At first glance, this could make you think it's the outer wrapper of the accordion, not a child of `Item.`

It's always nice to order elements here as close as possible to how they are nested.